Source/JavaScriptCore/API/JSWrapperMap.h

```diff
@@ -24,8 +24,8 @@
  */
 
 #import <JavaScriptCore/JavaScriptCore.h>
-#import <JSValueInternal.h>
 #import <objc/objc-runtime.h>
+#import "JSValueInternal.h"
```

Comment:
I think these need to be reordered to satisfy check-webkit-style.

Reply:
Done!